modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeHeader.scss

```diff
@@ -15,9 +15,24 @@ hr {
 		align-items: center;
 		display: flex;
 		justify-content: space-between;
+
+		.lfr__object-web-view-object-definitions-actions {
+			&-button {
+				align-items: center;
+				background-color: #fff;
+				color: #6b6c7e;
+				display: flex;
+				height: 24px;
+				justify-content: center;
+				margin-top: 0.5px;
+				padding: 0;
+				width: 24px;
+			}
+		}
```

Comment:
Hey @cgmonte, As this class has no dependency on `.lfr-objects__model-builder-node-header`, we can place it outside `.lfr-objects__model-builder-node-header`